Update endpoint-dlp-getting-started.md (#5093)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/endpoint-dlp-getting-started.md

	- For Windows 10 2004 - KB4568831, KB4577063
	- For devices running Office 2016 (and not any other Office version) - KB4577063 

4. All devices must be [Azure Active Directory (Azure AD) joined](/azure/active-directory/devices/concept-azure-ad-join), [Hybrid Azure AD joined](/azure/active-directory/devices/concept-azure-ad-join-hybrid), or [AAD registered](/azure/active-directory/user-help/user-help-register-device-on-network).

>    > [!NOTE] 

Reviewer: ```suggestion
> [!NOTE] 
```

>    > If using 'AAD registered' devices, then you must reboot the device after completing the AAD registration. This is required for each new user logging-in to the device.<br>

Reviewer: ```suggestion
   > - If you are using _Azure AD registered_ devices, you must reboot the device after completing the Azure AD registration. This is required for each new user signing in to the device.
   > - If you are using either _Azure AD registered_ or _Hybrid Azure AD joined_ devices, the device can be joined to on-premises Active Directory as well.
```

>    > If using either 'AAD registered' or 'Hybrid Azure AD joined' devices, then the device can be joined to on-premises Active Directory as well.<br>

Reviewer: ```suggestion
```
